SlackAPI/SlackTaskClient.cs

```diff
@@ -99,9 +99,20 @@ public Task<AuthTestResponse> TestAuthAsync()
             return APIRequestWithTokenAsync<AuthTestResponse>();
         }
 
-        public Task<UserListResponse> GetUserListAsync()
+        public Task<UserListResponse> GetUserListAsync(int limit = 0, bool include_locale = false, string cursor = null, string team_id = null)
         {
-            return APIRequestWithTokenAsync<UserListResponse>();
+            var args = new List<Tuple<string, string>>();
+            args.Add(new Tuple<string, string>("limit", limit.ToString()));
```

Comment:
Since limit is optional, this should probably be gated behind `limit > 0`, or at the very least the default `limit = 0` should be raised to a reasonable default (I'm thinking 100?)

Reply:
I agree with gating behind limit >= 0, but the default value of `limit` is 0, which mean slack will return as many entries as are present. I've tested this locally against my work's slack and `limit=0` does not return `0` entries, it returns all of them.
https://api.slack.com/methods/users.list#arg_limit